distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -89,6 +112,8 @@
 
     private static final String DEFAULT_SAMPLE_PRODUCT_FILE_NAME = "sample.txt";
 
+    private static final DynamicPort RESTITO_STUB_SERVER_PORT = new DynamicPort(41300);
```

Comment:
the number that gets passed to the new port is an offset from the `BASE_PORT` (which starts out at 20000)
So instead I recommend `new DynamicPort(6);` since we have 10 free slots to work with total, and 6 is the next one.
In fact, 20000 offset by 41300 is actually getting dangerously close to the max port (~65000)